docs/plugins/spotify.rst

```diff
@@ -1,23 +1,44 @@
 Spotify Plugin
 ==============
 
-The ``spotify`` plugin generates `Spotify`_ playlists from tracks in your library. Using the `Spotify Web API`_, any tracks that can be matched with a Spotify ID are returned, and the results can be either pasted in to a playlist or opened directly in the Spotify app.
+The ``spotify`` plugin generates `Spotify`_ playlists from tracks in your
+library with the ``beet spotify`` command. Using the `Spotify Search API`_,
+any tracks that can be matched with a Spotify ID are returned, and the
+results can be either pasted in to a playlist or opened directly in the
+Spotify app.
```

Comment:
Here's a slightly more direct way to say this without the passive voice: "Also, the plugin can use the Spotify Search API to provide metadata matches for the importer."

Reply:
This part of the doc is actually referring to the `beet spotify` command which uses the Search API, but good point about using the active voice. I've consolidated this with the portion below and it now reads:

> The `spotify` plugin generates Spotify playlists from tracks in your
library with the `beet spotify` command using the Spotify Search API.
Also, the plugin can use the Spotify Album and Track APIs to provide
metadata matches for the importer.

Not sure if we even need to specifically mention that we're using the Search/Album/Track APIs, I can remove that if not.